foodJournal/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from uuid import uuid4
+from django.contrib.auth.models import User
+
+# Create your models here.
+class Note(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    title = models.CharField(max_length=300)
+    ingredients = models.TextField(blank=True)
```

Comment:
maybe check this out if you wanted to have a list of predefined options for something like `meal` https://docs.djangoproject.com/en/2.1/ref/models/fields/#choices

Reply:
I will check it out. Thanks!